src/containerapp/azext_containerapp/_validators.py

```diff
@@ -21,7 +21,7 @@
 
 
 # called directly from custom method bc otherwise it disrupts the --environment auto RID functionality
-def validate_create(registry_identity, registry_pass, registry_user, registry_server, no_wait, source=None, repo=None, yaml=None, environment_type=None):
+def validate_create(registry_identity, registry_pass, registry_user, registry_server, no_wait, source=None, artifact=None, repo=None, yaml=None, environment_type=None):

```

Comment:
please add new argument behind existing arguments

Reply:
Generally, do not change the order of parameters.

Reply:
I wanted to keep --source and --artifact together as their behavior is very related to each other. Do you see this as a big problem? If yes I'm OK updating it to change the parameters location.